packages/plugin-ext/src/main/browser/view/tree-views-main.tsx

```diff
@@ -251,6 +259,32 @@ export class TreeViewWidget extends TreeWidget {
         });
     }
 
+    protected readonly onSelectionChangedEmitter = new Emitter<SelectionEventHandler>();
```

Comment:
always collect emitters!

Reply:
And place variable above the constructor